app/src/main/res/layout/activity_main.xml

```diff
@@ -29,6 +29,28 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/hellotext" />
 
+    <Button
```

Comment:
The buttons overlap the `GameActivity` button after you merged main
![image](https://github.com/SE2Project-BHKPTZ/frontend/assets/40315960/06b2a906-cd7e-4e75-80a8-d179c70d7537)